source/iterator/iterator.go

```diff
@@ -34,6 +34,8 @@ const (
 	metadataFieldTable = "redshift.table"
 	// keySearchPath is a key of get parameter of a datatable's schema name.
 	keySearchPath = "search_path"
+	// pingTimeoutSec is a ping db timeout in seconds.
+	pingTimeoutSec = 10
```

Comment:
Does this approach have some advantages over something like `pingTimeout = 10 * time.Second`? I mean, having the `pingTimeoutSec = 10` variable, we're forced to type `*time.Second` every time we use it. What do you think?

Reply:
You're damn right :boom:
Done!